docs/containers-registries.conf.5.md

```diff
@@ -109,6 +109,18 @@ internet without having to change `Dockerfile`s, or to add redundancy).
 *Note*: Redirection and mirrors are currently processed only when reading images, not when pushing
 to a registry; that may change in the future.
 
+#### Normalization of docker.io references
+
+The Docker Hub `docker.io` is handled in a special way, because every push and
+pull operation gets internally normalized with `/library` if no other specific
+namespace is defined (for example on `docker.io/username/image`).
```

Comment:
This feels a little opaque about what a “namespace” means.  I don’t think we should give a formal syntax here, changing the example to `docker.io/namespace/image` should be enough.

(Absolutely non-blocking: I might find it a little easier to read with the condition (“no namespace is explicitly specified”) before the consequence (“the image is normalized to use the default /library namespace”). )